src/commands/code-analyzer/run.ts

```diff
@@ -24,13 +24,15 @@ export default class RunCommand extends SfCommand<void> implements Displayable {
 		// === Flags pertaining to targeting ===
 		workspace: Flags.string({
 			summary: getMessage(BundleName.RunCommand, 'flags.workspace.summary'),
+			description: getMessage(BundleName.RunCommand, 'flags.workspace.description'),
```

Comment:
Why did these flags get descriptions here and not in `rules`?

Reply:
Good point!  If you think the same description in `run` works for `rules`, I'll add it.